In [2]:
import statistics
import math

population_mean = 120
sample_mean = 130.1
sample_std_dev = 21.21
sample_size = 100

z_test_statistic = (sample_mean - population_mean) / (sample_std_dev / math.sqrt(sample_size))

print("Z-test Statistic:", z_test_statistic)

Z-test Statistic: 4.761904761904759


Dado que 4.761904761904759 es mucho mayor que 1.96 (para α = 0.05), en este caso se rechazaría la hipótesis nula, lo que sugiere que la media de la muestra es significativamente diferente de la media de la población asumida.